pandas/tests/scalar/period/test_asfreq.py

```diff
@@ -791,14 +790,11 @@ def test_asfreq_combined(self):
 
     def test_asfreq_MS(self):
         initial = Period("2013")
```

Comment:
can you parameterize this test instead with these different cases

Reply:
I did, please have a look at it.